clippy_lints/src/methods/mod.rs

```diff
@@ -3632,6 +3633,28 @@ declare_clippy_lint! {
     "`as_str` used to call a method on `str` that is also available on `String`"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    /// Checks for usage of `clone().wake_by_ref()`
```

Comment:
```suggestion
    /// Checks for usage of `waker.clone().wake()`
```
I assume you meant to use `wake` here. (Also adding a receiver makes it clear that it's about `Waker`s)

Reply:
Thanks !